packages/cdk8s/test/yaml.test.ts

```diff
@@ -79,6 +79,17 @@ describe('save', () => {
     expect(fs.readFileSync(outputFile, 'utf-8')).toMatchSnapshot();
   });
 
+  test('strings are not folded', () => {
```

Comment:
Instead of this test, let's add a test which emulates the bad folding behavior you've seen.

Reply:
ok. like this?

```
  test('strings are not folded', () => {
    const longStringList = [
      '^(((\d*(\.\d*)?h)|(\d*(\.\d*)?m)|(\d*(\.\d*)?s)|' +
      '(\d*(\.\d*)?ms)|(\d*(\.\d*)?us)|(\d*(\.\d*)?µs)|' +
      '(\d*(\.\d*)?ns))+|infinity|infinite)$',
    ];
    const dumpPath = Yaml.tmp(longStringList);
    expect(fs.readFileSync(dumpPath, 'utf8').trimEnd()).not.toContain('\n');
  });
```

Reply:
Yes, exactly!